iroha/iroha.py

```diff
@@ -264,11 +264,13 @@ def query(self, name, counter=1, creator_account=None,
             created_time = self.now()
         if not creator_account:
             creator_account = self.creator_account
-        if page_size or first_tx_hash:
+        if page_size or first_tx_hash or ordering_sequence:
             pagination_meta = queries_pb2.TxPaginationMeta()
             pagination_meta.page_size = page_size
             if first_tx_hash:
                 pagination_meta.first_tx_hash = first_tx_hash
+            if ordering_sequence:
+                pagination_meta.ordering_sequence = ordering_sequence
```

Comment:
Here is problem:
```
AttributeError: 'TxPaginationMeta' object has no attribute 'ordering_sequence'
```
As I see [in documentation](https://iroha.readthedocs.io/en/main/develop/api/queries.html#result-pagination) it contains:
`ordering`, but when I try to change the name `ordering_sequence` -> `ordering`:
```
AttributeError: Assignment not allowed to field "ordering" in protocol message object.
```